File: PHP/Events/TicketInfo.php
<?php
namespace LTD_API;

class TicketInfo
{
    private $ticketId;
    private $ticketName1;
    private $ticketName2;
    private $isRestrictedView;
    private $restrictionDescription;
    
    public function getTicketId()
    {
        return $this->ticketId;
    }
    public function setTicketId($ticketId)
    {
        $this->ticketId = $ticketId;
    }

    public function getTicketName1()
    {
        return $this->ticketName1;
    }
    public function setTicketName1($ticketName1)
    {
        $this->ticketName1 = $ticketName1;
    }

    public function getTicketName2()
    {
        return $this->ticketName2;
    }
    public function setTicketName2($ticketName2)
    {
        $this->ticketName2 = $ticketName2;
    }

    public function getIsRestrictedView()
    {
        return $this->isRestrictedView;
    }
    public function setIsRestrictedView($isRestrictedView)
    {
        $this->isRestrictedView = $isRestrictedView;
    }

    public function getRestrictionDescription()
    {
        return $this->restrictionDescription;
    }
    public function setRestrictionDescription($restrictionDescription)
    {
        $this->restrictionDescription = $restrictionDescription;
    }
}

File: Java/Events/TicketInfo.java
package LTD_API;

public class TicketInfo
{
    private int ticketId;
    private String ticketName1;
    private String ticketName2;
    private boolean isRestrictedView;
    private String restrictionDescription;
    
    public int getTicketId()
    {
        return ticketId;
    }
    public void setTicketId(int ticketId)
    {
        this.ticketId = ticketId;
    }

    public String getTicketName1()
    {
        return ticketName1;
    }
    public void setTicketName1(String ticketName1)
    {
        this.ticketName1 = ticketName1;
    }

    public String getTicketName2()
    {
        return ticketName2;
    }
    public void setTicketName2(String ticketName2)
    {
        this.ticketName2 = ticketName2;
    }

    public boolean getIsRestrictedView()
    {
        return isRestrictedView;
    }
    public void setIsRestrictedView(boolean isRestrictedView)
    {
        this.isRestrictedView = isRestrictedView;
    }

    public String getRestrictionDescription()
    {
        return restrictionDescription;
    }
    public void setRestrictionDescription(String restrictionDescription)
    {
        this.restrictionDescription = restrictionDescription;
    }
}

